### PR TITLE
Upload image for use in blocknote editor

### LINKED ISSUE #1878: Ability to insert images within notes
See release here: https://github.com/TypeCellOS/BlockNote/releases/tag/v0.9.4

It's now possible to attach images within the Blocknote editor:
https://www.blocknotejs.org/docs/image-toolbar

On the backend we [already have](https://docs.twenty.com/graphql/) the `uploadImage` mutation, so it might not be that complex to implement!

### PR DESCRIPTION
**Pull request by brendanlaschke**
Completes the work of #1964 to also upload images.

Bildschirmfoto 2023-12-17 um 19 45 04

closes #1878

Fixed also local-storage not in gitignore

### CODE REVIEW

File: packages/twenty-front/src/modules/activities/components/ActivityBodyEditor.tsx [hunk 1]
```diff
@@ -56,6 +57,28 @@ export const ActivityBodyEditor = ({
...
+      variables: {
+        file,
+        fileFolder: FileFolder.Attachment,
+      },
+    });
+    if (!result?.data?.uploadFile) {
+      throw new Error("Couldn't upload Image");
+    }
+    const imageUrl =
+      process.env.REACT_APP_SERVER_BASE_URL +
```
Comment (charlesBochet) [inline packages/twenty-front/src/modules/activities/components/ActivityBodyEditor.tsx:76]: we should use: import { REACT_APP_SERVER_BASE_URL } from '~/config';
This allow run-time environment variable injection

File: packages/twenty-server/.gitignore [hunk 2]
```diff
@@ -1 +1,2 @@
-dist/*
\ No newline at end of file
+dist/*
+.local-storage
```
Comment (charlesBochet) [inline packages/twenty-server/.gitignore:2]: good point

**Review by charlesBochet — DISMISSED** [review]
LGTM with minor comment @brendanlaschke, thanks as always!

**Comment by charlesBochet** [comment]
@brendanlaschke I've found a bug actually. If the image is the first child of the body:
image


I think it's because it has no "content" section. We should update this logic:

```
  const body = JSON.parse(isNonEmptyString(task.body) ? task.body : '{}')[0]
    ?.content[0]?.text;
```

I also think we should extract it to an util `getActivitySummary` and cover it with jest tests to make sure it's behaving properly with task.body payload being: '{}', '', [...imageFirst], [...paragraphFirst], ... so this doesn't happen again

**Comment by brendanlaschke** [comment]
done - also for the similar preview in the notes tab ...

**Review by lucasbordeau — APPROVED** [review]

**Pull request merged**